controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/pkg/ApplicationPackage.java

```diff
@@ -202,7 +203,8 @@ private void preProcessAndPopulateCache() {
                                     new InputStreamReader(new ByteArrayInputStream(servicesXml.content()), UTF_8),
                                     InstanceName.defaultName(),
                                     Environment.prod,
-                                    RegionName.defaultName())
+                                    RegionName.defaultName(),
+                                    Tags.empty())
```

Comment:
Urgh, I guess this should preprocess with all configurations that may eventually be deployed.

Reply:
I don't know what this is for, hopefully it is just wasted work if there are variants?

Reply:
It is for storing all relevant deployment meta data to our audit store. If a certain override includes a file which isn't included in the default path taken here, that file will not be stored. It's not critical. Perhaps we could instead run only the include processor, and not the override one? Would that include all possible variants? (We don't care about the generated XML, but all files read by the inclusion process will be loaded to the file cache, which is then flushed to the audit storage.)

Reply:
You don't need to fix this in this PR.

Reply:
Ah, never mind, the include processor is run first.